sansio_lsp_client/structs.py

```diff
@@ -83,10 +94,20 @@ def calculate_length(self, text: str) -> int:
 class TextDocumentContentChangeEvent(BaseModel):
     text: str
     range: t.Optional[Range]
-    rangeLength: t.Optional[int]
+    rangeLength: t.Optional[int]  # deprecated, use .range
+
+    def dict(self) -> t.Dict[str, t.Any]:
+        d = super(TextDocumentContentChangeEvent, self).dict()
+
+        # vscode-css server requires un-filled values to be absent
+        if self.rangeLength is None:
+            del d["rangeLength"]
+        if self.range is None:
+            del d["range"]
+        return d
 
     @classmethod
-    def change_range(
+    def range_change(
```

Comment:
Why were `change_range` and `change_whole_document` renamed?

Reply:
* `change_range` - to change the range, i.e. modify instance
* `range_change` - a better name for method that creates new objects

same for the `change_whole_document`

If my english is failing me here and I'm wrong - will revert :)